spec.md

```diff
@@ -68,6 +68,17 @@ class Foo extends Bar implements FooInterface
         // method body
     }
 }
+
+Enum Beep: int
+{
+    case A = 1;
+    case B = 2;
```

Comment:
```suggestion
    case Foo = 1;
    case Bar = 2;
```

There are no difference between PascalCase and UPPER_CASE in the cases.
Please use more than one letter in a word.